skaffold/src/main/kotlin/com/google/container/tools/skaffold/SkaffoldExecutorService.kt

```diff
@@ -85,6 +85,8 @@ abstract class SkaffoldExecutorService {
         }
 
         try {
+            //this is where that whole no skaffold directory thing is happening
+            // Wait why isnt going into the catch
```

Comment:
oh it seems like this may not be ready yet for review? if so, its fine to put out the PR, just don't assign it for review until it is ready.